cvat/apps/engine/serializers.py

```diff
@@ -335,6 +335,13 @@ def update_label(
         parent_info, logger = cls._get_parent_info(parent_instance)
 
         attributes = validated_data.pop('attributespec_set', [])
+        encountered_names = set()
+        for attribute in attributes:
+            attr_name = attribute.get('name')
+            if attr_name in encountered_names:
+                raise exceptions.ValidationError('attribute with same name exists')
```

Comment:
Please, add attribute name for a better user feedback
Also, try to use `serializers.ValidationError` instead of `exceptions.ValidationError`